packages/config/src/hooks/afterResolveConfiguration.ts

```
@@ -13,6 +13,8 @@ export async function afterResolveConfiguration() {
 
   for (const source of sources.values()) {
     const {instance, watch, refreshOn, name, validationSchema} = source;
+    const seed = Math.floor(Math.random() * 10000);
+    const ref = name + "-" + seed;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)